__tests__/app.test.js

```diff
@@ -118,7 +118,49 @@ describe('app', () => {
             .then((response) => {
                 const responseMessage = response.body.msg
                 expect(responseMessage).toBe('invalid input id')
+            });
+        });
+    })
```

Comment:
We also want to add a test here (200) - that if the ID exists but there is no comments that it returns an empty array.